dependencies.yaml

```diff
@@ -66,12 +66,28 @@ dependencies:
               arch: x86_64
               cuda: "11.8"
             packages:
+              - cuda-version=11.8
               - nvcc_linux-64=11.8
+              - libcufile=1.4.0.31
+              - libcufile-dev=1.4.0.31
+              - libnvjpeg=11.6.0.55
+              - libnvjpeg-dev=11.6.0.55
```

Comment:
Not an absolute must, but it would be nice to split this into a separate matrix entry for all cuda 11.8 (x86 and arm) since half of these are common.

Reply:
Ah actually this doesn't work. dfg doesn't (yet) support multiple matches with different subsets of the matrix. Never mind!

Reply:
Oh interesting! Ok should we file an issue here to follow up?

Reply:
Filed as issue ( https://github.com/rapidsai/cucim/issues/585 ). Please add more details there